datastream-bigquery/src/main/java/com/linkedin/datastream/bigquery/translator/RecordTranslator.java

```diff
@@ -177,8 +177,12 @@ private static Object translateMapTypeObject(Object record, Schema avroSchema) {
             case BYTES:
                 if (LogicalTypeIdentifier.isDecimalType(avroSchema)) {
                     final LogicalTypes.Decimal decimalType = (LogicalTypes.Decimal) avroSchema.getLogicalType();
-                    result = new AbstractMap.SimpleEntry<>(name,
-                            new BigDecimal(new BigInteger(((ByteBuffer) record).array()), decimalType.getScale()));
+                    BigDecimal scaledBigDecimal = new BigDecimal(new BigInteger(((ByteBuffer) record).array()), decimalType.getScale());
```

Comment:
Can we move these conversion code to a static method like convertBytesToDecimal() with some comments on Scale handle?  maybe in a common translator class? so there is no duplicates even it's small.

Reply:
- I added the test result in the PR description above.
- It's a good idea to move this to a common utils class but we should do this after Chris is done with his changes and when we refactor the BigQuery translator into the common translator.

Reply:
Okay. just keep in mind we need to do this at some point.